src/Outlet.ts

```diff
@@ -30,7 +30,7 @@ export type Outlet<
 
 export function Outlet<W extends WidgetBaseInterface, F extends WidgetBaseInterface, E extends WidgetBaseInterface>(
 	outletComponents: Component<W> | OutletComponents<W, F, E>,
-	outlet: string,
+	outlet: string | Array<String>,
```

Comment:
We use `[]` for array types, `string[]` - Also should avoid using `String` and use `string` instead (https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).

Reply:
Done, thanks for explaining, I will read that page. Can we lint for this sort of thing?

Reply:
@umaar https://palantir.github.io/tslint/rules/ban-types/ so it appears yes... let's open an issue on https://github.com/dojo/dojo2-package-template where the _master_ `tslint.json` file resides.  We likely need to do a wholesale review.

Reply:
[Issue raised!](https://github.com/dojo/dojo2-package-template/issues/95)